src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Fixtures/DataImportGeneratorExtensions.cs

```diff
@@ -12,55 +12,140 @@ public static Generator<DataImport> DefaultDataImport(this DataFixture fixture)
     public static Generator<DataImport> WithDefaults(this Generator<DataImport> generator)
         => generator.ForInstance(d => d.SetDefaults());
     
-    public static InstanceSetters<DataImport> SetDefaults(this InstanceSetters<DataImport> setters)
-        => setters
-            .SetDefault(d => d.Id)
-            .SetFiles("data");
-    
     public static Generator<DataImport> WithSubjectId(
         this Generator<DataImport> generator,
         Guid subjectId)
-        => generator.ForInstance(s => s.Set(d => d.SubjectId, subjectId));
+        => generator.ForInstance(s => s.SetSubjectId(subjectId));
     
-    public static Generator<DataImport> WithFiles(
+    public static Generator<DataImport> WithDefaultFiles(
         this Generator<DataImport> generator,
         string dataFileName)
-        => generator.ForInstance(s => s.SetFiles(dataFileName));
+        => generator.ForInstance(s => s.SetDefaultFiles(dataFileName));
+
+    public static Generator<DataImport> WithFile(
+        this Generator<DataImport> generator,
+        File file)
+        => generator.ForInstance(s => s.SetFile(file));
+
+    public static Generator<DataImport> WithMetaFile(
+        this Generator<DataImport> generator,
+        File metaFile)
+        => generator.ForInstance(s => s.SetMetaFile(metaFile));
+
+    public static Generator<DataImport> WithZipFile(
+        this Generator<DataImport> generator,
+        File zipFile)
+        => generator.ForInstance(s => s.SetZipFile(zipFile));
 
     public static Generator<DataImport> WithStatus(
         this Generator<DataImport> generator,
         DataImportStatus status)
-        => generator.ForInstance(s => s.Set(d => d.Status, status));
+        => generator.ForInstance(s => s.SetStatus(status));
+
+    public static InstanceSetters<DataImport> WithStagePercentageComplete(
+        this InstanceSetters<DataImport> setters,
+        int stagePercentageComplete)
+        => setters.Set(d => d.StagePercentageComplete, stagePercentageComplete);
```

Comment:
The body of this looks like it should be named `SetStagePercentageComplete`, but then we are missing `WithStagePercentageComplete`.